### ISSUE TITLE
documentation needs to have complete list of required Python modules

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#81]
There's a list at https://github.com/CCPPETMR/SIRF/wiki/SIRF-and-Python, but it is incomplete as for testing we need `nose` and `coverage` as well. The OS-specific pages have their own list of Python modules, which also misses those 2. It's unfortunate to have the duplication but seems unavoidable.

Are there any others missing?

Naming for conda: `nose` and `coverage`, for apt-get on Ubuntu `python-nose` and `python-coverage`, pip seems to be `nose` and `codecov` (from .travis.yml).

@paskino, can you check those are installed on the VM?

**Comment by paskino** [comment]
on the VM we install
```python-scipy 
python-docopt  
python-numpy 
python-h5py 
python-matplotlib 
python-libxml2 
python-psutil 
spyder 
python-tk
```

**Comment by KrisThielemans** [comment]
`python-libxml2` and `python-psutil` would only be useful for Gadgeton Python gadgets as far as I know (and we're not using them). I don't think `h5py` and `python-tk` are required either

**Comment by KrisThielemans** [comment]
updated the wiki. hopefully fine now. there's probably too many listed, but that's better than not enough... we will fix this better wtih #124
